content/en/docs/reference/config-api/kubelet-config.v1beta1.md

```diff
@@ -1637,6 +1637,13 @@ option is explicitly enabled.
 Default: false</p>
 </td>
 </tr>
+<tr><td><code>userNamespaces</code><br/>
+<a href="#kubelet-config-k8s-io-v1beta1-UserNamespaces"><code>UserNamespaces</code></a>
+</td>
+<td>
+   <p>UserNamespaces contains config for user namespaces</p>
```

Comment:
...contains for the pod user namespace?

To differentiate of other user namespaces, like the kubelet in a user namespaces, etc.?

Reply:
done